sqlgen/db.go

```diff
@@ -361,6 +361,52 @@ func (db *DB) InsertRow(ctx context.Context, row interface{}) (sql.Result, error
 	return db.execWithTrace(ctx, query, "InsertRow")
 }
 
+// InsertRows inserts multiple rows into the database, chunksize rows at a time.
+// Most SQL db enforce a limit on max size of packet, which is why we need to break
+// the rows into chunks.
+//
+// rows should be an array of pointers to a struct, for example:
+//
+//   user1 := &User{Name: "foo"}
+//   user2 := &User{Name: "fan"}
+//   if err := db.InsertRows(ctx, []{interface{}}{user1, user2}, 100); err != nil {
+//
+func (db *DB) InsertRows(ctx context.Context, rows []interface{}, chunkSize int) (sql.Result, error) {
```

Comment:
@stephen do you have a preference on taking in `[]interface{}` or `interface{}` and walking with reflect (like [this](https://golang.org/pkg/sort/#Slice))? if this takes `[]interface{}`, the caller may have to convert a strongly typed slice to an interface typed slice in a loop, because `[]*User` is not assignable to `[]interface{}`.

Reply:
I agree that interface{} would be better. changed